spec/support/features/session_helper.rb

```diff
@@ -690,5 +690,9 @@ def fill_reset_password_form(without_request_id: nil)
 
       expect(current_path).to eq new_user_session_path
     end
+
+    def expect_branded_experience
+      expect(page).to have_css('.page-header--basic img ~ img')
```

Comment:
could we add a `data-` or some other attribute to the class we care about? this seems harder to explain/find

Reply:
Yeah, I played with a few different ideas with this. Ideally I'd have liked to look for the image by name, but since it could vary based on service provider, that's a little trickier. At the very least I could add the "Login.gov" logo alt text to try to clarify that we're looking for "the Login.gov logo next to the SP logo". I'm a bit reluctant to introduce runtime markup changes to the common layout just for the tests.

Reply:
yeah alt text on one of the images would be clearer!

Reply:
Updated in 6c04f9336. I also finding a good way to incorporate the partner logo alt text in there as well, but that's hard to do without requiring it be explicitly passed as a parameter.